### PR TITLE
{math}[GCC/8.3.0] lpsolve v5.5.2.5

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login1.cerberus.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a1f1081d75e76599c3989c5ad8931b9c for a full test report.

File: easybuild/easyconfigs/l/lpsolve/lpsolve-5.5.2.5-GCC-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['lp_solve_%(version)s_source.tar.gz']
+patches = ['lpsolve-%(version)s_Fix_expected_identifier_error_for_gcc.patch']
+checksums = [
+    '201a7c62b8b3360c884ee2a73ed7667e5716fc1e809755053b398c2f5b0cf28a',  # lp_solve_5.5.2.5_source.tar.gz
+    # lpsolve-5.5.2.5_Fix_expected_identifier_error_for_gcc.patch
+    'be46d06035797b455ff616c74f83c4e36c9cad6c41b0c8680b29cb6b3208f49c',
+]
+
+dependencies = [('binutils', '2.32')]
```
Comment (smoors) [inline easybuild/easyconfigs/l/lpsolve/lpsolve-5.5.2.5-GCC-8.3.0.eb]: @lexming binutils is already a dependency of GCC, so you can remove it.

Comment (lexming) [reply easybuild/easyconfigs/l/lpsolve/lpsolve-5.5.2.5-GCC-8.3.0.eb]: I didn't realize that `GCC/8.3.0` also includes `binutils` :open_mouth: 
Fixed :+1:

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login1.cerberus.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/83b15b04415282c44d16e97df7518492 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**